app/models/job_proxy_dispatcher.rb

```diff
@@ -13,6 +13,7 @@ def initialize
 
   def dispatch
     _dummy, t = Benchmark.realtime_block(:total_time) do
+      Benchmark.realtime_block(:container_dispatching) {dispatch_container_jobs}
```

Comment:
How long does this method take?  I think the schedule worker dumps dispatch on the queue every [15 seconds by default](https://github.com/ManageIQ/manageiq/blob/1e8fa2e1b52c5f57fbdffb3096ce9ab84d3dc6f0/config/settings.yml#L1363).  Is this frequency too often for containers?